packages/eslint-config/index.js

```diff
@@ -1,3 +1,52 @@
 module.exports = {
-  extends: 'airbnb-base',
-};
+    extends: 'airbnb-base',
+    rules: {
+        'arrow-parens': ['error', 'as-needed'],
+        'comma-dangle': ['error', 'only-multiline'],
+        'eol-last': 'off',
+        /*
+            This is the default value which is defined here to note that there may be exceptions
+            where it is appropriate to disable this rule for older projects.
+        */
+        'func-names': ['error', 'as-needed'],
+        indent: ['error', 4],
+        'linebreak-style': 'off',
+        /*
+            This is the default value which is defined here to note that there may be exceptions
+            where it is appropriate to disable this rule for older projects or with a comment.
+            Including one class per file is a best practice and recommended by the Angular style
+            guide. However, migrating older projects may not be trivial, and there may be
+            exceptions for organization.
+        */
+        'max-classes-per-file': ['error', 1],
+        /*
+            This is the default value which is defined here to note that there may be exceptions
+            where it is appropriate to disable this rule with a comment. Awaits in loops are
+            expected to be mistakes more often than not. However, there may be exceptions for
+            unique situations.
```

Comment:
Those situations should be documented, right? _This_ file _is_ the NI style guide, which should give guidance. (Or we could link to a markdown file somewhere with additional examples?)